api_data.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from urllib.request import urlopen
+
+def output_cve_id(type=None, name=None, version=None):
+    """Take as input, a package name, package version.
```

Comment:
I had a talk with @pombredanne regarding this. 
This is the way he suggested it to be. 
We can discuss this further?  :-)